python/paddle/v2/fluid/initializer.py

```diff
@@ -132,6 +132,8 @@ def __call__(self, var, block):
         assert isinstance(var, framework.Variable)
         assert isinstance(block, framework.Block)
         # Initialization Ops should be prepended and not appended
+        if block.program.random_seed != 0 and self._seed == 0:
```

Comment:
If we don't consider case that user need to overwrite program default rand seed with 0, just use `if self._seed == 0` is fine, maybe?

Reply:
that's true.